client/v2/algod/tealDisassemble.go

```diff
@@ -0,0 +1,23 @@
+package algod
+
+import (
+	"context"
+
+	"github.com/algorand/go-algorand-sdk/client/v2/common"
+	"github.com/algorand/go-algorand-sdk/client/v2/common/models"
+)
+
+// TealDisassemble given the base64 encoded program bytes, return the TEAL source
+// code in plain text. This endpoint is only enabled when a node's configuration
+// file sets EnableDeveloperAPI to true.
+type TealDisassemble struct {
+	c *Client
+
+	source string
```

Comment:
Yes, there are two things wrong:
- The description should not mention base64, that's a problem with go-algorand: https://github.com/algorand/go-algorand/blob/aa82b9a6e78dd38c8115120ca90b817f6be73ef3/daemon/algod/api/algod.oas2.json#L1441
- This field should be `[]byte` since it's declared with `"format": "byte"` in the API spec. There may be an issue with the generator since that's not the case.

Since the TEAL disassembly has nothing to do with this PR, I'd suggest manually removing all references to it from the generated files and we can address them later.